toonz/sources/toonz/menubar.cpp

```diff
@@ -669,6 +669,7 @@ void TopBar::loadMenubar() {
   addMenuItem(windowsMenu, MI_OpenComboViewer);
   addMenuItem(windowsMenu, MI_OpenLevelView);
   addMenuItem(windowsMenu, MI_OpenFileViewer);
+  addMenuItem(windowsMenu, MI_OpenVectorInspectorPanel);
```

Comment:
Please move this new entry down to the bottom section of the Panels menu list. This isn't an actual image viewer and is more a utility feature.

Reply:
Completed.